test/api.test.js

```diff
@@ -0,0 +1,25 @@
+const supertest = require('supertest');
+const should = require('should');
+
+const server = supertest.agent('http://localhost:3333');
+
+describe('API unit test', function() {
+
+  //should return home page
+
+  it('should return home page', function(done) {
+    //calling home page api
+    server
+      .get('/')
+      .expect('Content-type', /json/)
```

Comment:
Like this setup.  Going to add the JSON test as well

Reply:
Thanks!